packages/iov-core/src/jsrpc.ts

```diff
@@ -0,0 +1,242 @@
+import {
+  jsonRpcCodeInternalError,
+  jsonRpcCodeInvalidParams,
+  jsonRpcCodeInvalidRequest,
+  jsonRpcCodeMethodNotFound,
+  jsonRpcCodeParseError,
+  jsonRpcCodeServerErrorDefault,
+  SimpleMessagingConnection,
+} from "@iov/jsonrpc";
+import { firstEvent } from "@iov/stream";
+
+// An RPC interface very similar to JSON-RPC but supporting
+// all the data types from the structured clone algorithm
+// https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Structured_clone_algorithm#Supported_types
+
+/** A single structured clone algorithm compatible value */
+export declare type JsRpcCompatibleValue =
```

Comment:
This is nice and good. Types plus link above adds clarity, changelog was more confusing